src/main/java/se/bjurr/violations/lib/example/MyClass.java

```diff
@@ -30,6 +31,6 @@ void npe2(String a, String b) {
 
  @Override
  public boolean equals(Object obj) {
-  return true;
+  return false;
  }
```

Comment:
**Reporter**: Findbugs
**Rule**: EQ_ALWAYS_TRUE
**Severity**: ERROR
**File**: se/bjurr/violations/lib/example/MyClass.java L35
**Source**: se.bjurr.violations.lib.example.MyClass

equals method always returns true&#10;&#10;&#10;  &lt;p&gt; This class defines an equals method that always returns true. This is imaginative, but not very smart.&#10;Plus, it means that the equals method is not symmetric.&#10;&lt;/p&gt;&#10;


*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1266990394>*